posts/2022-09-22-Rust-1.64.0.md

```diff
@@ -0,0 +1,260 @@
+---
+layout: post
+title: "Announcing Rust 1.64.0"
+author: Josh Triplett
+team: The Release Team <https://www.rust-lang.org/governance/teams/release>
+release: true
+---
+
+The Rust team is happy to announce a new version of Rust, 1.64.0. Rust is a
+programming language empowering everyone to build reliable and efficient
+software.
+
+If you have a previous version of Rust installed via rustup, you can get 1.64.0
+with:
+
+```console
+rustup update stable
+```
+
+If you don't have it already, you can [get
+`rustup`](https://www.rust-lang.org/install.html) from the appropriate page on
+our website, and check out the [detailed release notes for
+1.64.0](https://github.com/rust-lang/rust/blob/stable/RELEASES.md#version-1640-2022-09-22)
+on GitHub.
+
+If you'd like to help us out by testing future releases, you might consider
+updating locally to use the beta channel (`rustup default beta`) or the nightly
+channel (`rustup default nightly`). Please
+[report](https://github.com/rust-lang/rust/issues/new/choose) any bugs you
+might come across!
+
+## What's in 1.64.0 stable
+
+### C-compatible FFI types in core and alloc
+
+When calling or being called by C ABIs, Rust code can use type aliases like
+`c_uint` or `c_ulong` to match the corresponding types from C on any target,
+without requiring target-specific code or conditionals.
+
+Previously, these type aliases were only available in `std`, so code written
+for embedded targets and other scenarios that could only use `core` or `alloc`
+could not use these types.
+
+Rust 1.64 now provides all of the `c_*` type aliases in
+[`core::ffi`](https://doc.rust-lang.org/core/ffi/index.html), as well as
+[`core::ffi::CStr`](https://doc.rust-lang.org/core/ffi/struct.CStr.html) for
+working with C strings. Rust 1.64 also provides
+[`alloc::ffi::CString`](https://doc.rust-lang.org/alloc/ffi/struct.CString.html)
+for working with owned C strings using only the `alloc` crate, rather than the
+full `std` library.
+
+### New internal layout for network structures
```

Comment:
I would move this down the post: it's in the blog mostly for compatibility, not because it has a high user-facing impact.